src/pages/idle-users.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC, useState, useEffect } from 'react';
+import Head from '@/components/head';
+import Section from '@/components/idleUsers/section';
+import Layout from '@/components/Layout';
+import useFetch from '@/hooks/useFetch';
+import { IDLE_USERS_URL } from '@/components/constants/url';
+import { idleUser } from '@/interfaces/idleUser.type';
+
+const IdleUsers: FC = () => {
+  const [idleUsersList, setIdleUsersList] = useState<idleUser[]>([]);
+
+  const {
+    response,
+    error,
+    isLoading,
+  } = useFetch(IDLE_USERS_URL);
+
+  useEffect(() => {
+    if ('allUserStatus' in response) {
+      const idleUsers = response.allUserStatus;
+      setIdleUsersList(idleUsers);
+    }
+  }, [isLoading, response]);
```

Comment:
why are we adding isLoading as dependency?

Reply:
Because we have created a custom hook called useFetch and it returns us `isLoading`.